assets/app/scripts/services/logLinks.js

```diff
@@ -11,12 +11,26 @@ angular.module('openshiftConsole')
       // TODO (bpeterse): a lot of these functions are generic and could be moved/renamed to
       // a navigation oriented service.
 
+      var breakpoint = {
+        'mobile': 768
+      };
+
       var scrollTop = function() {
-        $window.scrollTo(null, 0);
+        if($(window).width() < breakpoint.mobile) {
+          $window.scrollTo(null, 0);
+        } else {
+          var objDiv = $(".middle-container.has-scroll")[0];
+          objDiv.scrollTop = 0;
+        }
```

Comment:
This looks like where we did a quick fix to get something working.  I agree, we should stabilize this.